src/components/primitives/Option/Checkbox.md

```diff
@@ -7,13 +7,13 @@ import ThemeTester from "../../../styleguide/ThemeTester";
 
 <React.Fragment>
   <ThemeTester>
-    <Text>checked Checkbox without label</Text>
+    <Text style={{marginBottom: 0}}>checked Checkbox without label</Text>
     <Checkbox name="c3" checked />
-    <Text>Checkbox without label</Text>
+    <Text style={{marginBottom: 0}}>Checkbox without label</Text>
     <Checkbox name="c4" />
-    <Text>Checkbox with primitive label</Text>
+    <Text style={{marginBottom: 0}}>Checkbox with primitive label</Text>
     <Checkbox name="c5" label="dummy label" />
-    <Text>Checkbox with sophisticated label</Text>{" "}
+    <Text style={{marginBottom: 0}}>Checkbox with sophisticated label</Text>{" "}
```

Comment:
still not perfect though... we need to wrap each input with text in `div` but I suggest to make it in new PR

Reply:
add `props` to Text that define marginLess Text/Title

Reply:
@qunabu maybe `noMargin` is a more appropriate name?

Reply:
both names works for me

Reply:
> add `props` to Text that define marginLess Text/Title

I added a new props to the `<Text>` component because <Title> has a `margin 0` so it doesn't really make sense